src/main/webapp/resources/js/pages/projects/linelist/components/Table/LoadingOverlay.jsx

```diff
@@ -1,18 +1,55 @@
-import React, { Component } from "react";
+import React from "react";
+import { Progress, Space, Typography } from "antd";
+import { Component } from "react";
+import { connect } from "react-redux";
+import { grey5 } from "../../../../../styles/colors";
 
-import { Spin } from "antd";
-
-export class LoadingOverlay extends Component {
+/**
+ * React component for AG GRID to display a loading overlay
+ */
+class LoadingOverlay extends Component {
   render() {
     return (
-      <div className="ag-overlay-loading-center irida-ag-overlay">
-        <div>
-          <Spin style={{ marginRight: "1rem" }} />
-          <span className="irida-ag-overlay-loading-text">{`${i18n(
-            "linelist.agGrid.loading"
-          )}`}</span>
-        </div>
+      <div
+        style={{
+          backgroundColor: "#FFFFFF",
+          boxShadow: `rgba(0, 0, 0, 0.16) 0px 1px 4px`,
+          borderRadius: "2px",
+          border: `1px solid ${grey5}`,
+          display: "flex",
+          justifyContent: "center",
+          alignItems: "center",
+          height: "100%",
+        }}
+      >
+        <Space
+          style={{
+            width: `100%`,
+            height: 300,
+            padding: 20,
+            display: "flex",
+            justifyContent: "center",
+            alignItems: "center",
+            flexDirection: "column",
+          }}
+        >
+          <Progress
+            percent={Math.floor(
```

Comment:
Move this calculation into a function

Reply:
Really does not need to be

Reply:
Updated in 792c9c565c2fd496f034e075806eb56a3d4772a1